docs/transparent-validation/sonoff/quick-start-guide.md

```diff
@@ -0,0 +1,160 @@
+# Sonoff documentation
+
+## Quick start guide
+
+The below documentation is based on the
+[EPSHome documentation](https://esphome.io/devices/sonoff_s20.html)
+
+1. Open the Sonoff case.
+
```

Comment:
Is there an option to add image for this section?

Reply:
There are very precise instructions with photos in link above (line 6)

Reply:
Updated